opensync-ext-cloud/src/main/java/com/telecominfraproject/wlan/opensync/external/integration/OpensyncExternalIntegrationMqttMessageProcessor.java

```diff
@@ -1670,19 +1670,36 @@ void populateApNodeMetrics(List<ServiceMetric> metricRecordList, Report report,
                 busySelf += surveySample.getBusySelf() * surveySample.getDurationMs();
                 totalDurationMs += surveySample.getDurationMs();
                 noiseList.add(getNegativeSignedIntFrom8BitUnsigned(surveySample.getNoise()));
-
             }
 
             if (totalDurationMs > 0) {
                 RadioUtilization radioUtil = new RadioUtilization();
                 radioUtil.setTimestampSeconds((int) ((survey.getTimestampMs()) / 1000));
                 int pctBusyTx = busyTx / totalDurationMs;
+                if (pctBusyTx > 100 || pctBusyTx < 0) {
+                	LOG.error("WIFI-1237 survey.getBand {}, survey.getTimestampMs {} \n survey.getSurveyListList {} ",
```

Comment:
Please change the log method to LOG.warn and remove "WIFI-1237" from the log message. Also do not use \n in the log message - it will be difficult to parse. The message should be something like "calculated value for ___ is out of bounds .... "